Buck changes for ObjC/Swift bindings.

shoumikhin opened the pull request:
Summary: Add resources for the tests.

Differential Revision: D70377502

kirklandsign commented:
From my understanding, we want to export a model every time we run the test, unless we are doing BC/FC check.

shoumikhin commented:
@kirklandsign ptal at https://github.com/pytorch/executorch/tree/main/extension/module/test/resources, we kinda replicate the same setup here

shoumikhin committed:
Buck changes for ObjC/Swift bindings. (#8825)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/8825

Add resources for the tests.

Reviewed By: mergennachin

Differential Revision: D70377502
